new-components/targets/git-clone/go.mod

```diff
@@ -0,0 +1,88 @@
+module github.com/smithy-security/smithy/new-components/targets/git-clone
+
+go 1.23.2
+
+require (
+	github.com/go-errors/errors v1.5.1
+	github.com/go-git/go-billy/v5 v5.5.0
+	github.com/go-git/go-git/v5 v5.12.0
+	github.com/jonboulle/clockwork v0.4.0
+	github.com/ory/dockertest/v3 v3.11.0
+	github.com/smithy-security/pkg/env v0.0.0-20241121114636-0915d48f6d71
```

Comment:
Need to merge and tag  https://github.com/smithy-security/pkg/pull/1and update this dep.

Reply:
Done